doc/api/packages.md

```diff
@@ -547,6 +542,47 @@ exports, while resolving the existing `"node"`, `"default"`, `"import"`, and
 
 Any number of custom conditions can be set with repeat flags.
 
+### Conditions Definitions
+
+The `"import"`, `"require"`, `"node"` and `"default"` conditions are defined
+and implemented in Node.js core, [as specified above](#conditional_exports).
+
+The following condition definitions are currently endorsed by Node.js:
+
+* `"browser"` - any environment which implements a standard subset of
```

Comment:
This should probably indicate it has zero runtime implications?

Reply:
Can you clarify what you mean by this?

Reply:
“endorses” certainly just means conceptually, but the other 2 conditions in this list actually do something in node, while “browser” doesn’t. Deleting it from this list wouldn’t impact node code whatsoever - so i think it’d be useful to clarify, in this list item, that it’s a convention, but that node doesn’t pay attention to it in any way.

Reply:
This whole list is of conditions that do not have implementations in node. That is, in theory you could set `--conditions=browser` when running Node.js to achieve the same effect. The condition definitions in this section are thus entirely by convention, and not enforceable by Node.js.

If we had a `--dev` or `--production` flag or assertion of mutual exclusivity we could possibly move the `"development"` and `"production"` conditions into the above section which are the implemented conditions.

Does that separation make sense? In that this whole list is meant to be this distinction?

Reply:
the description in the next two describes how to make node behave differently for the respective condition - this one doesn’t. Why are all three not described similarly?

Reply:
It's only by convention that we can stop people from doing `--conditions=browser`, as unless you are running eg JSDom that does not typically apply to Node.js. `--conditions=production` and `--conditions=development` on the other hand are explicitly encouraged patterns.

Reply:
Previously we discussed having eg a `--dev` or `--production` flag set both the condition and `process.env.NODE_ENV`. Such a pattern might justify moving these up to the "implementation" section, as then it's not simply a convention at that point.

Reply:
In that case, why is `browser` mentioned at all?

In other words, I think it's fine to mention "how you use `--conditions=` with production/development", if that's what we want to encourage. However, if "browser" is mentioned, then we should either tell people how to use it with `--conditions=`, or, we should tell people *not* to, and detail how it's intended to be a convention for bundlers to follow, like the existing top-level browser field (which has a spec we may want to link to).

Reply:
> In that case, why is browser mentioned at all?

Because browser is being used, and there is no formal definition for it. Node.js docs can be a home for these definitions or not. If you disagree then let's move this somewhere else.

Reply:
I think there’s a long list of conditions that will be used without a formal definition. I think it’s a slippery slope to include them in node’s docs at all, but if included, they should be very clearly called out as nothing more than documented convention.

Reply:
That's exactly what this entire section of the documentation is supposed to be - documented convention only. How can I make that clearer?

Reply:
development and production have instructions of how to use them in node, browser does not. Either all 3 should, or all 3 shouldn’t.

Reply:
Ok, see the latest commit - I've moved the `--conditions` note to be generic for the section.

Reply:
Awesome, thanks, that resolves my concerns!